composer.json

```diff
@@ -45,10 +34,22 @@
     },
     "bin": ["bin/coveralls"],
     "scripts": {
+        "install-devtools": [
+            "cd devtools && composer update"
+        ],
         "sca": [
-            "php vendor/bin/php-cs-fixer fix --dry-run -vv",
-            "php vendor/bin/phpcs --report=checkstyle --report-file=build/logs/checkstyle.xml --standard=build/config/phpcs.xml --ignore=*.html.php,*.config.php,*.twig.php src",
-            "php vendor/bin/phpmd src xml build/config/phpmd.xml --reportfile build/logs/pmd.xml"
+            "php devtools/vendor/bin/php-cs-fixer fix --dry-run -vv",
+            "php devtools/vendor/bin/phpcs --report=checkstyle --report-file=build/logs/checkstyle.xml --standard=build/config/phpcs.xml --ignore=*.html.php,*.config.php,*.twig.php src",
+            "php devtools/vendor/bin/phpmd src xml build/config/phpmd.xml --reportfile build/logs/pmd.xml"
+        ],
+        "apigen": [
+            "php devtools/vendor/bin/apigen -c build/config/apigen.neon -d build/api -s src -s vendor/symfony/config -s vendor/symfony/console -s vendor/symfony/yaml -s vendor/guzzle/guzzle/src -s vendor/psr/log --report build/logs/checkstyle-apigen.xml --exclude *Test.php --exclude */Tests/* --google-analytics UA-40398434-1"
+        ],
+        "box": [
+            "mkdir -p build/artifacts",
+            "composer update --no-dev",
+            "php -d phar.readonly=false ${BOX_BIN} build",
```

Comment:
If box works from the devtools/vendor directory, that's great, but it doesn't work because of autoloader detection failure. Now assuming phar/global installation of box and you can set the BOX_BIN envvar to point to your installation.